superset/charts/api.py

```diff
@@ -105,9 +99,6 @@ def ensure_thumbnails_enabled(self) -> Optional[Response]:
         RouteMethod.IMPORT,
         RouteMethod.RELATED,
         "bulk_delete",  # not using RouteMethod since locally defined
-        "data",
```

Comment:
I move them to ChartDataRestApi to avoid duplications and mental confusion because that class should not aware that other are inherited it